batoms/logger.py

```diff
@@ -0,0 +1,46 @@
+"""
+Logger
+"""
+import bpy
+import sys
+from tempfile import gettempdir
+from pathlib import Path
+import logging
+import os
+
+batoms_dir = os.path.dirname(Path(__file__).parent)
+
+
+
+# logger = logging.getLogger('batoms')
+logger = logging.getLogger(__name__)
+root_logger = logging.getLogger("batoms")
+
+def set_logger(version):
+    # formatter = ('%(levelname)-8s '
+    #                             '[%(funcName)-20s]: %(message)s')
+    formatter = ('%(levelname)s '
```

Comment:
@superstar54 I modified the logger output so module & func name are both shown. Please modify as you prefer